indico/modules/events/timetable/client/js/actions.ts

```diff
@@ -143,27 +144,26 @@ export function setDraftEntry(data): SetDraftEntryAction {
   return {type: SET_DRAFT_ENTRY, data};
 }
 
-export function moveEntry(entry, eventId, entries: TopLevelEntry[], date: string) {
-  let entryURL: string;
-
-  switch (entry.type) {
-    case EntryType.Break:
-      entryURL = breakURL({event_id: eventId, break_id: entry.objId});
-      break;
-    case EntryType.SessionBlock:
-      entryURL = sessionBlockURL({event_id: eventId, session_block_id: entry.objId});
-      break;
-    default:
-      entryURL = contributionURL({event_id: eventId, contrib_id: entry.objId});
-  }
-
-  const entryData = {start_dt: moment(entry.startDt).format('YYYY-MM-DDTHH:mm:ss')};
-
-  return synchronizedAjaxAction(() => indicoAxios.patch(entryURL, entryData), {
-    type: MOVE_ENTRY,
-    date,
-    entries,
-  });
+export function moveEntry(entry, eventId, entries: TopLevelEntry[], date: string, sessionBlockId?) {
+  return dispatch => {
+    const newEntry = {...entry};
+
+    const entryURL = getEntryURL(newEntry, eventId);
+
+    const entryData = {
+      start_dt: moment(newEntry.startDt).format('YYYY-MM-DDTHH:mm:ss'),
```

Comment:
I think there should be a cleaner way to do this than formatting manually with a hardcoded string